/*
 * Copyright (c) Huawei Technologies Co., Ltd. 2025. All rights reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

syntax = "proto3";

package lease_service;

import "src/libruntime/fsclient/protobuf/common.proto";

option go_package = "faas-pattern/pkg/common/grpc/pb/lease;lease";

message LeaseRequest {
  string    remoteClientId = 1;
}
message LeaseResponse {
  common.ErrorCode code    = 1;
  string           message = 2;
}
